addons/website_event/views/event_templates_list.xml

```diff
@@ -160,51 +137,127 @@
 <!-- Filter - Date -->
 <template id="event_time" inherit_id="website_event.index_topbar" name="Filter by Date">
     <xpath expr="//div[hasclass('o_wevent_search')]" position="before">
-        <div class="dropdown d-none d-lg-block me-2 my-1">
-            <a href="#" role="button" class="btn btn-light dropdown-toggle" data-bs-toggle="dropdown">
+        <div class="dropdown d-none d-lg-block">
+            <a href="#" role="button" class="btn btn-light dropdown-toggle" data-bs-toggle="dropdown" title="Filter by Date">
                 <t t-if="current_date" t-out="current_date"/>
                 <t t-else="">Upcoming Events</t>
             </a>
             <div class="dropdown-menu">
                 <t t-foreach="dates" t-as="date">
                     <t t-if="date[3] or (date[0] in ('old','upcoming','all'))">
-                        <a t-att-href="keep('/event', date=date[0])" t-attf-class="dropdown-item d-flex align-items-center justify-content-between #{searches.get('date') == date[0] and 'active'}">
+                        <t t-set="isActive" t-value="searches.get('date') == date[0]"/>
```

Comment:
Nitpick but it would be better to follow the python scheme in python templates
```suggestion
                        <t t-set="is_active" t-value="searches.get('date') == date[0]"/>
```

same everywhere else

Reply:
not addressed yet